### PR TITLE
Swope changes ocm

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: governance/create_gatekeeper.adoc [hunk 1]
```diff
@@ -175,7 +175,7 @@ Gatekeeper policy, Gatekeeper, and Gatekeeper operator policy are uninstalled.
 == Additional resources
 
 - See xref:../governance/gatekeeper_policy.adoc#gatekeeper-policy[Integrating Gatekeeper constraints and constraint templates] for details about Gatekeeper.
-- See the link:https://github.com/stolostron/policy-collection/blob/main/community/CM-Configuration-Management/policy-gatekeeper-operator.yaml[`policy-gatekeeper-operator.yaml`] sample.
+- See the link:https://github.com/open-cluster-management/policy-collection/blob/main/community/CM-Configuration-Management/policy-gatekeeper-operator.yaml[`policy-gatekeeper-operator.yaml`] sample.
```
Comment (dockerymick) [inline governance/create_gatekeeper.adoc]: I recommend that we use this link instead: https://github.com/open-cluster-management-io/policy-collection/blob/main/stable/CM-Configuration-Management/policy-gatekeeper-operator-downstream.yaml

Comment (dockerymick) [inline governance/create_gatekeeper.adoc]:
```suggestion
- See the link:https://github.com/open-cluster-management-io/policy-collection/blob/main/stable/CM-Configuration-Management/policy-gatekeeper-operator-downstream.yaml[`policy-gatekeeper-operator-downstream.yaml`] sample.
```

File: governance/grc_intro.adoc [hunk 2]
```diff
@@ -24,7 +24,7 @@ The governance architecture is composed of the following components:
...
 
-* *Open source community:* Supports community contributions with a foundation of the {product-title-short} policy framework. Policy controllers and third-party policies are also a part of the link:https://github.com/stolostron/policy-collection[`stolostron/policy-collection` repository]. You can contribute and deploy policies using GitOps. For more information, see _Deploying policies by using GitOps_ in the _Manage security policies_ section. Learn how to integrate third-party policies with {product-title}.
+* *Open source community:* Supports community contributions with a foundation of the {product-title-short} policy framework. Policy controllers and third-party policies are also a part of the link:https://github.com/open-cluster-management/policy-collection[`stolostron/policy-collection` repository]. You can contribute and deploy policies using GitOps. For more information, see _Deploying policies by using GitOps_ in the _Manage security policies_ section. Learn how to integrate third-party policies with {product-title}.
```
Comment (dockerymick) [inline governance/grc_intro.adoc]:
```suggestion
* *Open source community:* Supports community contributions with a foundation of the {product-title-short} policy framework. Policy controllers and third-party policies are also a part of the link:https://github.com/open-cluster-management/policy-collection[`open-cluster-management/policy-collection` repository]. You can contribute and deploy policies using GitOps. For more information, see _Deploying policies by using GitOps_ in the _Manage security policies_ section. Learn how to integrate third-party policies with {product-title}.
```

File: governance/kyverno_add_network_policy.adoc [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [#kyverno-add-network-policy]
 = Kyverno add network policy
 
-The Kyverno add network policy configures a new `NetworkPolicy` resource called `default-deny` which denies all traffic when you create a new namespace. You must install Kyverno controller to use the Kyverno policy. See link:https://github.com/stolostron/policy-collection/blob/main/community/CM-Configuration-Management/policy-install-kyverno.yaml[`policy-install-kyverno.yaml`] for the install policy.
+The Kyverno add network policy configures a new `NetworkPolicy` resource called `default-deny` which denies all traffic when you create a new namespace. You must install Kyverno controller to use the Kyverno policy. See link:https://github.com/open-cluster-management/policy-collection/blob/main/community/CM-Configuration-Management/policy-install-kyverno.yaml[`policy-install-kyverno.yaml`] for the install policy.
```
Comment (dockerymick) [inline governance/kyverno_add_network_policy.adoc]: I did only see this in the community folder. @swopebe I remember that we said we wanted to point to stable policy examples. @mprahl is there another policy sample that we can provide?

File: governance/kyverno_add_quota_policy.adoc [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [#kyverno-add-quota-policy]
 = Kyverno add quota policy
 
-The Kyverno add quota policy configures new `ResourceQuota` and `LimitRange` resources when you create a new namespace. You must install Kyverno controller to use the Kyverno policy. See link:https://github.com/stolostron/policy-collection/blob/main/community/CM-Configuration-Management/policy-install-kyverno.yaml[`policy-install-kyverno.yaml`] for the install policy.
+The Kyverno add quota policy configures new `ResourceQuota` and `LimitRange` resources when you create a new namespace. You must install Kyverno controller to use the Kyverno policy. See link:https://github.com/open-cluster-management/policy-collection/blob/main/community/CM-Configuration-Management/policy-install-kyverno.yaml[`policy-install-kyverno.yaml`] for the install policy.
```
Comment (dockerymick) [inline governance/kyverno_add_quota_policy.adoc]: same comment

File: governance/kyverno_sync_secrets_policy.adoc [hunk 5]
```diff
@@ -1,7 +1,7 @@
 [#kyverno-sync-secrets-policy]
 = Kyverno sync secrets policy
 
-The Kyverno sync secrets policy copies a secret called `regcred` which exists in the `default` namespace to new namespaces you create and updates the secret when changes are detected. You must install Kyverno controller to use the Kyverno policy. See link:https://github.com/stolostron/policy-collection/blob/main/community/CM-Configuration-Management/policy-install-kyverno.yaml[`policy-install-kyverno.yaml`] for the install policy.
+The Kyverno sync secrets policy copies a secret called `regcred` which exists in the `default` namespace to new namespaces you create and updates the secret when changes are detected. You must install Kyverno controller to use the Kyverno policy. See link:https://github.com/open-cluster-management/policy-collection/blob/main/community/CM-Configuration-Management/policy-install-kyverno.yaml[`policy-install-kyverno.yaml`] for the install policy.
```
Comment (dockerymick) [inline governance/kyverno_sync_secrets_policy.adoc]: same comment about the community link

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: governance/kyverno_add_network_policy.adoc (hunk 3 above)
Comment (swopebe) [reply governance/kyverno_add_network_policy.adoc]: Agreed, @mprahl we either need to talk to PM to see if this should be part of the doc/product or remove this? @berenss do you have input on _install Kyverno controller to use the Kyverno policy_ (add to the official doc or remove link from official doc, as the doc should only be what goes through the feature>code>qe process.)

Comment (swopebe) [reply governance/kyverno_add_network_policy.adoc]: This link is in other areas of the doc.

Comment (mprahl) [reply governance/kyverno_add_network_policy.adoc]: Yes, that policy is broken and is a community policy that shouldn't be documented.

Comment (berenss) [reply governance/kyverno_add_network_policy.adoc]: Hi Brandi thank you - Looks like @ch-stark would be better aligned to comment here (and perhaps this is already resolved)

**Review by dockerymick — APPROVED** [review]
Links look good, thanks for this update!

**Comment by ch-stark** [comment]
@dockerymick yesterday we decided in the Call to remove Kyverno-Policies from documentation.  Can we handle this here?  I can open a new Bug if needed

**Comment by swopebe** [comment]
@ch-stark as part of the Done criteria for all the dev teams, they are required to review deprecations and removals. They do not open bugs, but they open tasks with all the references that need to be removed.

In this case, if it was added incorrectly and never used by the customer, we do not need to deprecate or add to the table, we just need to get it out of the doc because it causes problems.

Yes to a task from the dev team, which @dockerymick can bring up in their scrum.

**Comment by dockerymick** [comment]
@swopebe thanks for confirming. Here is the related issue https://issues.redhat.com/browse/ACM-7893
I will go ahead and remove the Kyverno policies or we can reassign if necessary. I think it is still worth it to update the removals table to describe why it was removed

**Comment by swopebe** [comment]
Well, if it was a mistake, I would rather we just remove it and not put it in the table, as that table is for active components that the customer is using. 

@dockerymick let's see if anyone has shown it to a RHACM subscriber or if it has intentionally been used, or if it just should not have been added. 

@ch-stark was this in production in the product or only mentioned in the doc?

**Comment by swopebe** [comment]
I am gonna merge this bc I have to move on, we can cont the discussion in the new issue.

**Comment by dockerymick** [comment]
@swopebe @ch-stark confirmed. Here is his response: 

"Mikela Jackson we never Supported Kyverno really, we just Support the PolicyGenerator part

We thought we might support it one time, this is why we created the Policies and wanted to make it official but it makes not so much sense as we do not provide a supported way to install Kyverno

just removing the 3 Policies is fine"

**Pull request merged**